code/game/machinery/lightswitch.dm

```diff
@@ -16,6 +16,7 @@
 
 /obj/machinery/light_switch/initialize()
 	add_self_to_holomap()
+	toggle_switch(0)
```

Comment:
I could understand what you meant by this code with a little bit of thinking but it's not obvious. Could always add a comment explaining that you force the light switch to `off` by adding `0` as an argument.

Reply:
another way you could make this obvious without a lengthy comment might be to include the arg name, e.g.: `toggle_switch(newstate = 0)` which I think is fairly obvious in its meaning